crates/storage/db/src/tables/mod.rs

```diff
@@ -406,6 +406,9 @@ tables! {
 
     /// Stores EIP-7685 EL -> CL requests, indexed by block number.
     table BlockRequests<Key = BlockNumber, Value = Requests>;
+
+    /// Stores the last finalized block.
+    table FinalizedBlocks<Key = u8, Value = BlockNumber>;
```

Comment:
if we name the table this way, we won't be able to reuse it for anything else. pls consider giving some generic name

Reply:
makes sense, renamed to `ChainState`, ptal